src/configure.rs

```diff
@@ -70,7 +71,14 @@ pub fn print_configuration(use_default: bool) {
 
     let string_config = toml::to_string_pretty(&config).unwrap();
 
-    println!("# Warning: This config does not include keys that have an unset value");
+    println!("# Warning: This config does not include keys that have an unset value\n");
+    println!(
+        "# $all is shorthand for {}",
+        PROMPT_ORDER
+            .iter()
+            .map(|module_name| format!("${}", module_name))
```

Comment:
For custom modules, you need `${custom.whatever}`.

Reply:
I have it also written as 
```
# empty line on purpose to get a empty line
format = """

${custom.toolbox}\
$username\
$hostname\
...
"""

Reply:
The original code didn't actually have any handling for `$custom`. If changed the code, so that if any custom modules are in the config `$custom` is expanded in addition to `$all`.

Regarding printing the modules line-by-line I wanted to avoid that, because it would make the comment take too much space.